### PR TITLE
[docker logging plugin] More cleanup

### PR DESCRIPTION
**Pull request by fearful-symmetry**
This is another round of cleanup to standardize naming and clean up the readme.

### CODE REVIEW

File: x-pack/dockerlogbeat/readme.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# dockerlogbeat: a simple, a docker plugin for sending logs to elasticsearch
+# Elastic Logging Plugin: a docker plugin for sending logs to elasticsearch
 
 
-This code is a working alpha for a [docker logging plugin](https://docs.docker.com/engine/extend/plugins_logging/). With the proper config, it can send logs to elasticsearch.
+This code is a working MVP for a [docker logging plugin](https://docs.docker.com/engine/extend/plugins_logging/). With the proper config, it can send logs to elasticsearch.
 ## Build and install
 
-To build and install, just run `mage create`
+To build and install, just run `mage Package`
```
Comment (kaiyan-sheng) [inline x-pack/dockerlogbeat/readme.md]: mage package?

File: x-pack/dockerlogbeat/magefile.go [hunk 2]
```diff
@@ -97,6 +97,6 @@ func Install() error {
 }
 
 // Create builds and creates a docker plugin
```
Comment (urso) [inline x-pack/dockerlogbeat/magefile.go]: Package builds and create a docker logging plugin

File: x-pack/dockerlogbeat/readme.md (hunk 1 above)
Comment (fearful-symmetry) [reply x-pack/dockerlogbeat/readme.md]: Yah, I changed the name to be more consistent with the rest of the beats.

Comment (exekias) [reply x-pack/dockerlogbeat/readme.md]: I think what Kaiyan meant was you to use lowercase here, as this is how all beats expose this in mage help (btw both upper and lower cases seem to work). For example:

```
X1 :: elastic/beats/metricbea » mage                                                                                                                                                                              2019/11/25 15:53:19 Found Elastic Beats dir at /home/exekias/go/src/github.com/elastic/beats
Targets:
  ...
  package                           packages the Beat for distribution.
  ...
```

**Pull request merged**